client/appchannel_test.go

```diff
@@ -55,8 +55,7 @@ func TestAppChannel(t *testing.T) {
 		),
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), TestTimeoutBlocks*s.BlockTime)
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
```

Comment:
Same here. Similar to the TwoPartyTestTimeout in go-perun?

Reply:
I don't want to use a global timeout. Each test has a different running time.